docs/src/pages/components/masonry/BasicMasonry.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable @typescript-eslint/no-use-before-define */
+import * as React from 'react';
+import Box from '@material-ui/core/Box';
+import Masonry from '@material-ui/lab/Masonry';
+import MasonryItem from '@material-ui/lab/MasonryItem';
+
+export default function BasicMasonry() {
+  return (
+    <Masonry columns={4} spacing={1}>
+      {heights.map((height, index) => (
+        <MasonryItem key={index}>
```

Comment:
Shouldn't we apply the height here for SSR?
```suggestion
        <MasonryItem key={index} height={height}>
```

Reply:
Yes, I think it is a good idea that we optionally receive `height` prop for users who want SSR.

Reply:
In fact, now I think about it, if user passes a content whose height (e.g. 80px) is different from the height the user passes (e.g. 100px) to `MasonryItem`, there will be unwanted gap.